content_scripts/desktopAdapter.js

```diff
@@ -139,6 +139,14 @@ createNameSpace('realityEditor.device.desktopAdapter');
                     color: env.groundWireframeColor,
                     thickness: 0.075 // relatively thick
                 });
+                // todo Steve: make ai chatbox turn on dark mode
+                let aiContainer = document.getElementById('ai-chat-tool-container');
+                let searchTextArea = [...aiContainer.querySelectorAll('#searchTextArea')][0];
```

Comment:
This is equivalent to the more straightforward `aiContainer.querySelector('#searchTextArea')`